qa/tasks/scripts.py

```diff
@@ -19,8 +19,17 @@ def __init__(self, ctx, logger):
             ctx['scripts_copied'] = True
 
     def run(self, remote, script_name, args=[], as_root=True):
-        self.log.info('(scripts) running script {} on remote {}'
-                      .format(script_name, remote.hostname)
+        remote_type = type(remote)
+        self.log.debug(
+            '(scripts) run method was passed a remote object of type {}'
+            .format(remote_type)
+            )
```

Comment:
in the test log, this comes out as `(scripts) run method was passed a remote object of type <class 'teuthology.orchestra.cluster.Cluster'>`